packages/core-data/src/selectors.js

```diff
@@ -280,9 +280,7 @@ export const __experimentalGetDirtyEntityRecords = createSelector(
 								entityRecord[
 									entity.key || DEFAULT_ENTITY_KEY
 								],
-							title: ! entity.getTitle
-								? ''
-								: entity.getTitle( entityRecord ),
+							title: entity?.getTitle( entityRecord ) || '',
```

Comment:
This is just a small refactor but actually is another place that we could handle the change as here, `entityRecord ` has the title as well.

Right above `entityRecord` is retrieved from this:
```
const entityRecord = getEditedEntityRecord(
	state,
	kind,
	name,
	primaryKey
);
```

Another alternative would be to search and pass all the required fields, such as `title` to get `rendered` to `getEditedEntityRecord`. That would mean changing the signature of more functions and doesn't seem right to me.